### ISSUE TITLE
[HOLD #41254][$250] IOS - After editing split expense amount weird flash and reloads at the top

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#41174]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.67-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @roryabraham 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1714161321481359

## Action Performed:

1. Create a group
2. Add messages to the chat such that there is at least one full screen of chat messages.
3. Scan a receipt to split a bill in the group.
4. Manually set the amount on the bill split

## Expected Result:
The amount should update, and nothing else should happen.
## Actual Result:
The amount updates, then the report "flashes" and reloads at the top, then scrolls down to the bottom, without you touching anything.
## Workaround:
unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [x] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/38435837/82bec8e2-0c90-42c9-9b47-4b398a09bc7f

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0174f92c2a5d39efca</li>
        <li>Upwork Job ID: 1784657619645915136</li>
        <li>Last Price Increase: 2024-05-05</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @roryabraham</details>

**Comment by eh2077** [comment]
Posted on [slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1714550741651229) to attract proposals

**Comment by eh2077** [comment]
Not overdue, still waiting for proposals

**Comment by eh2077** [comment]
@charles-liang Thanks for your proposal. Your RCA is correct and I agreed with you this issue should be fixed by the backend.
`StartSplitBill` returns the correct `previousReportActionID`
<img width="845" alt="image" src="https://github.com/Expensify/App/assets/117511920/b056ecb9-87dd-4b2c-8150-2828342fbd44">
While `CompleteSplitBill` returns wrong `previousReportActionID`.
<img width="824" alt="image" src="https://github.com/Expensify/App/assets/117511920/07ce0950-1c4c-4604-b36e-f41c9e579703">

🎀👀🎀 C+ reviewed

**Comment by roryabraham** [comment]
ok, I keep seeing a number of issues with incorrect `previousReportActionID` generated in the back-end

**Comment by eh2077** [comment]
I think @roryabraham will take care of this from the backend. Once the backend fix is done, we can retest it to determine if it's good to close the issue.

**Comment by arielgreen** [comment]
@roryabraham can you confirm if you're going to be working on this or if it needs a hot pick label?

**Comment by roryabraham** [comment]
I'm still working on this, though it won't necessarily be back-end. I'm writing up a thorough P/S in [this doc](https://docs.google.com/document/d/19xEEr0mY41cTio1lGgJ2F6vYCA5QH4Sj7hOMRZsj0Dc/edit)

**Comment by roryabraham** [comment]
Requested retest: https://expensify.slack.com/archives/C9YU7BX5M/p1715988615903839

**Comment by kavimuru** [comment]
Bug is still able to reproduce.

https://github.com/Expensify/App/assets/43996225/6b0da52e-bf4a-458b-a4eb-6504d7c5dc77

**Comment by saracouto** [comment]
@roryabraham moved the status of this bug to in progress since you are working on it. since the priority is high, i am gonna make this a daily!

**Comment by roryabraham** [comment]
I spent prettymuch the whole day working on this problem [over here](https://expensify.slack.com/archives/C05LX9D6E07/p1716305457352729?thread_ts=1714656403.682529&cid=C05LX9D6E07), lots and lots of context in there 😅

**Comment by roryabraham** [comment]
Going to put this on HOLD for https://github.com/Expensify/App/issues/41254 the sake of consolidation. This has been my top priority, and we've been actively working on it as you can see in the slack thread. But I have like 4 different GitHub issues under the same ☂️ and they haven't all been getting the same love

**Comment by roryabraham** [comment]
still "on HOLD", still prettymuch my top priority (along with all other issues in the same ☂️)

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by roryabraham** [comment]
HOLDing, we have recent activity in https://github.com/Expensify/App/pull/41962

**Comment by laurenreidexpensify** [comment]
@sonialiap another parental leave one - there is no action on this required this week, it's on hold and @roryabraham is working on it. Thanks!

**Comment by sonialiap** [comment]
I'm OOO Aug 19-30, but this is monthly so not adding a BZ leave buddy
